webapp/src/ts/modules/contacts/contacts-content.component.ts

```diff
@@ -46,20 +48,31 @@ export class ContactsContentComponent implements OnInit, OnDestroy {
     this.subscribeToChanges();
     this.setReportsTimeWindowMonths(3);
     this.setTasksTimeWindowWeeks(1);
+    this.getUserFacility();
   }
 
   ngOnDestroy() {
     this.subscription.unsubscribe();
   }
 
+  getUserFacility() {
```

Comment:
Can this be private?
This applies to all the methods in this class that are not intended to be used outside of its scope.

I find it's a lot clearer when the API that you're intending to expose is well established. Would you mind making this change?

Reply:
No, don't mind. I will make the change